localstack/services/s3/constants.py

```diff
@@ -55,6 +63,18 @@
 # TODO: potentially look into making configurable
 ENABLE_MOTO_BUCKET_POLICY_ENFORCEMENT = False
 
+
+SYSTEM_METADATA_SETTABLE_HEADERS = [
+    "CacheControl",
+    "ContentDisposition",
+    "ContentEncoding",
+    "ContentLanguage",
+    "ContentMD5",
+    "ContentType",
+]
+
+STREAM_CHUNK_SIZE = 2048 * 2
```

Comment:
is this something defined by S3 or something that we came up with? in that case, we could maybe clarify why it's that value. also, were you too lazy to calculate that it's 4096 ? :laughing:

Reply:
I think it's not even used 😅 will remove